Kernel/Boot/CommandLine.cpp

```diff
@@ -79,7 +79,7 @@ UNMAP_AFTER_INIT CommandLine::CommandLine(StringView cmdline_from_bootloader)
 
 Optional<StringView> CommandLine::lookup(StringView key) const
 {
-    return m_params.get(key);
+    return static_cast<Optional<StringView>>(m_params.get(key));
```

Comment:
As this happens quite often, I wonder if we should just set the `PeekType` of `StringView` and/or other classes to by-value instead

Reply:
Absolutely! Span and (NN)RP are the same